tests/env/test_create.py

```diff
@@ -27,229 +26,218 @@ def get_env_vars(prefix):
 
 
 @pytest.mark.integration
-def test_create_update(conda_cli: CondaCLIFixture, monkeypatch: MonkeyPatch):
-    with make_temp_envs_dir() as envs_dir:
-        monkeypatch.setenv("CONDA_ENVS_DIRS", envs_dir)
-        reset_context()
-        assert context.envs_dirs[0] == envs_dir
-
-        env_name = str(uuid4())[:8]
-        prefix = Path(envs_dir, env_name)
-
-        conda_cli(
-            *("env", "create"),
-            *("--name", env_name),
-            *("--file", support_file("example/environment_pinned.yml")),
-        )
-        assert prefix.exists()
-        assert package_is_installed(prefix, "python")
-        assert package_is_installed(prefix, "flask=2.0.2")
-
-        env_vars = get_env_vars(prefix)
-        assert env_vars["FIXED"] == "fixed"
-        assert env_vars["CHANGES"] == "original_value"
-        assert env_vars["GETS_DELETED"] == "not_actually_removed_though"
-        assert "NEW_VAR" not in env_vars
-
-        stdout, stderr, err = conda_cli(
-            *("env", "update"),
-            *("--name", env_name),
-            *("--file", support_file("example/environment_pinned_updated.yml")),
-        )
-        PrefixData._cache_.clear()
-        assert package_is_installed(prefix, "flask=2.0.3")
-        assert not package_is_installed(prefix, "flask=2.0.2")
-
-        env_vars = get_env_vars(prefix)
-        assert env_vars["FIXED"] == "fixed"
-        assert env_vars["CHANGES"] == "updated_value"
-        assert env_vars["NEW_VAR"] == "new_var"
-
-        # This ends up sticking around since there is no real way of knowing that an environment
-        # variable _used_ to be in the variables dict, but isn't any more.
-        assert env_vars["GETS_DELETED"] == "not_actually_removed_though"
+def test_create_update(
+    conda_cli: CondaCLIFixture,
+    monkeypatch: MonkeyPatch,
+    tmp_envs_dir: Path,
+):
+    env_name = uuid4().hex[:8]
+    prefix = tmp_envs_dir / env_name
 
+    conda_cli(
+        *("env", "create"),
```

Comment:
I understand this syntax was here before, but what's the point of doing this? Isn't it basically the same as doing:

```python
conda_cli(
    "env", "create",
    "--name", env_name,
    "--file", support_file("example/environment_pinned.yml")
)
```

Reply:
it was done this way since ruff/black would reformat it:

```python
conda_cli(
    "env",
    "create",
    "--name",
    env_name,
    "--file",
    support_file("example/environment_pinned.yml"),
)
```

which is harder to read since it is unclear that the arguments belong to the flag

this style was adopted and used before I realized that `argparse` supports `=`, so the preferred style is now:

```python
conda_cli(
    "env",
    "create",
    f"--name={env_name}",
    f"--file={support_file('example/environment_pinned.yml')}",
)
```

but I deemed doing that here was out of scope